src/features/DebugSession.ts

```diff
@@ -105,7 +114,7 @@ export class PickPSHostProcessFeature implements IFeature {
 
         if (this.waitingForClientToken) {
             this.clearWaitingToken();
-            return this.pickPSHostProcess();
```

Comment:
BTW if we want to punt on this for 0.9.0, I could return an error message when PowerShell hasn't been started along the lines of `"You must start a PowerShell session before using attach to PowerShell host process"`.